charts/nginx-gateway-fabric/values.yaml

```diff
@@ -13,14 +24,24 @@ nginxGateway:
   # -- Set of custom annotations for GatewayClass objects.
   gatewayClassAnnotations: {}
 
+  # @schema
+  # pattern: ^gateway.nginx.org/.*
+  # required: true
+  # @schema
   # -- The name of the Gateway controller. The controller name must be of the form: DOMAIN/PATH. The controller's domain
   # is gateway.nginx.org.
   gatewayControllerName: gateway.nginx.org/nginx-gateway-controller
 
   # The dynamic configuration for the control plane that is contained in the NginxGateway resource.
   config:
     logging:
-      # -- Log level. Supported values "info", "debug", "error".
+      # @schema
+      # enum:
+      #   - info
+      #   - debug
+      #   - error
+      # @schema
+      # -- Log level.
```

Comment:
Is there a reason to not include the supported values comment, that removes it from the chart readme?

Reply:
I don't think it adds much value, they pretty standard options and they're documented here through the schema anyway. Also with this you'll get autocomplete ands suggestions in your editor.